### PR TITLE
Add to team wizard - add phone numbers

### PR DESCRIPTION
**Pull request by buoyad**
image

image

^ Single line phone input on mobile ticketed separately. cc @keybase/y2ksquad

### CODE REVIEW

File: shared/common-adapters/button.tsx [hunk 1]
```diff
@@ -27,12 +27,10 @@ export type ButtonColor = 'blue' | 'red' | 'green' | 'purple' | 'black' | 'yello
 type WithIconProps =
   | {
       icon?: never
-      tooltip?: string
     }
   | {
       icon: IconType
       iconColor?: Styles.Color
-      tooltip: string
```
Comment (jakob223) [inline shared/common-adapters/button.tsx]: Perhaps go through and delete the places where we used `tooltip=''` to get around this if we're removing this rule?

Do you know why it was there in the first place? a11y? Something else?

File: shared/signup/phone-number/phone-input.tsx [hunk 2]
```diff
@@ -238,6 +238,7 @@ type Props = {
   defaultCountry?: string
```
Comment (jakob223) [inline shared/signup/phone-number/phone-input.tsx]: does this file want to move to common adapters?

File: shared/teams/add-members-wizard/add-from-where.tsx [hunk 3]
```diff
@@ -22,6 +22,8 @@ const AddFromWhere = (props: Props) => {
 
   const onClose = () => dispatch(nav.safeNavigateUpPayload())
   const onContinueKeybase = () => dispatch(appendNewTeamBuilder(teamID))
+  const onContinuePhone = () =>
```
Comment (jakob223) [inline shared/teams/add-members-wizard/add-from-where.tsx]: I feel like this isn't really useful to have without also putting the data in the store. But I guess that's happening in a future ticket anyway so /shrug

**Comment by jakob223** [comment]
Heh I found a fun bug: if you delete all the phone number inputs, you can't add any more.

File: shared/common-adapters/button.tsx (hunk 1 above)
Comment (buoyad) [reply shared/common-adapters/button.tsx]: Done :+1: not sure why it was required.

File: shared/teams/add-members-wizard/add-from-where.tsx (hunk 3 above)
Comment (buoyad) [reply shared/teams/add-members-wizard/add-from-where.tsx]: I have some of the store setup started in https://github.com/keybase/client/pull/22957.

**Comment by buoyad** [comment]
Ah good catch @jakob223, fixed by not allowing removing the last phone number.

File: shared/signup/phone-number/phone-input.tsx (hunk 2 above)
Comment (buoyad) [reply shared/signup/phone-number/phone-input.tsx]: ya

Comment (buoyad) [reply shared/signup/phone-number/phone-input.tsx]: done

**Pull request merged**